x-pack/auditbeat/module/system/host/host.go

```diff
@@ -315,11 +323,59 @@ func hostEvent(host *Host, eventType string, action eventAction) mb.Event {
 				"type":   eventType,
 				"action": action.String(),
 			},
+			"message": hostMessage(host, action),
 		},
 		MetricSetFields: host.toMapStr(),
 	}
 }
 
+func hostMessage(host *Host, action eventAction) string {
+	// Hostname + IP of the first non-loopback interface.
+	hostString := fmt.Sprintf("%v (IP: %v)", host.info.Hostname, ipString(host.addrs[0]))
+
+	var message string
+	switch action {
+	case eventActionHost:
+		message = fmt.Sprintf("%v host %v is up for %v",
+			host.info.OS.Name, hostString, fmtDuration(host.uptime))
+	case eventActionIDChanged:
+		message = fmt.Sprintf("ID of host %v has changed", hostString)
```

Comment:
Didn't realize that could happen. Can you report previous & new ID?

Reply:
Yeah it can, I've been testing it with `rm -f /etc/machine-id && dbus-uuidgen --ensure=/etc/machine-id`.

The code detects this, and will send two events:
- One will contain all the old host information, and a `system.audit.host.new_id` field with the new host ID (while the old one is still in `system.audit.host.id`.
- The other will contain all the new host information, and a `system.audit.host.old_id` field with the old host ID (while the new one is already in `system.audit.host.id`.

That way, it is possible to match the two together and get a full continuous picture.

Reply:
Great!